docs: add guide to generate a container with scss (#1652)
ngrx/platform: projects/ngrx.io/content/guide/schematics/container.md

```sh
ng generate container UsersPage --state reducers/index.ts --stateInterface MyState
```

> If you want to generate container with scss file, add `@ngrx/schematics:container` to schematics in `angular.json` as bellow.

Reviewer: ```suggestion
If you want to generate a container with an scss file, add `@ngrx/schematics:container` to the `schematics` in your `angular.json`.
```

Author: thank you. I applied it.
